### PR TITLE
Testing out cljfmt on 2.3.x

### PR DESCRIPTION
**Pull request by senior**
Pushing up this PR more for discussion than anything else. There's two calls that I used, `lein cljfmt check` which just returns a diff of the changes that it would make and `lein cljfmt fix` which made the changes that I included in the second commit. Probably best to discuss this verbally vs. text. Below is some info for the discussion.

- The formatting in this PR is just stock cljfmt, I didn't make any changes.
- Checking the formatting `lein cljfmt check` runs in about 80 seconds on my laptop
- `lein cljfmt fix` also runs in about 80 seconds, a little longer if it needs to make changes

The cljfmt page is [here](https://github.com/weavejester/cljfmt).

### CODE REVIEW

File: src/com/puppetlabs/puppetdb/cli/anonymize.clj [hunk 1]
```diff
@@ -18,161 +18,108 @@
...
-            "Augeas" "Computer" "Cron" "Exec" "File" "Filebucket" "Group" "Host"
-            "Interface" "K5login" "Macauthorization" "Mailalias" "Mcx" "Mount"
-            "Notify" "Package" "Resources" "Router" "Schedule" "Schedule_task"
-            "Selboolean" "Selmodule" "Service" "Ssh_authorized_key" "Sshkey" "Stage"
-            "Tidy" "User" "Vlan" "Yumrepo" "Zfs" "Zone" "Zpool"]}
-           "anonymize" false}
-          {"context" {"type" "/^Nagios_/"} "anonymize" false}
+  {"full" {;; Full anonymization means anonymize everything
+           "rules" {}}
+   "moderate" {"rules" {"type" [;; Leave the core type names alone
```
Comment (ajroetker) [inline src/com/puppetlabs/puppetdb/cli/anonymize.clj]: Interesting that it reformatted this to one line

**Comment by ajroetker** [comment]
I definitely don't want to be in charge of this rebase

File: src/com/puppetlabs/puppetdb/cli/anonymize.clj (hunk 1 above)
Comment (mullr) [inline src/com/puppetlabs/puppetdb/cli/anonymize.clj]: That's contrary to standard lisp commenting rules too. `;;` indicates a comment relating to the next line, while `;` is used for an end-of-line comment

File: src/com/puppetlabs/http.clj [hunk 2]
```diff
@@ -133,15 +132,15 @@
...
-       (log/debug error "Caught HTTP processing exception")
-       (-> msg
-           (rr/response)
-           (rr/status code)))))
+   (let [msg (if (instance? Throwable error)
+               (.getMessage error)
+               (str error))]
+     (log/debug error "Caught HTTP processing exception")
+     (-> msg
+         (rr/response)
```
Comment (ajroetker) [inline src/com/puppetlabs/http.clj]: I wish this also removed unnecessary parentheses

File: src/com/puppetlabs/http.clj [hunk 3]
```diff
@@ -161,16 +160,16 @@
...
-     (let [segments (uri-segments uri)
-           f        (fn [[segs strs] u]
-                      (let [segs' (conj segs u)]
-                        [segs'
-                         (conj strs (str delimiter (s/join delimiter segs')))]))]
-       (second (reduce f [[] []] segments)))))
+   {:pre  [(.startsWith uri "/")]
+    :post [(coll? %)]}
+   (let [segments (uri-segments uri)
+         f        (fn [[segs strs] u]
```
Comment (ajroetker) [inline src/com/puppetlabs/http.clj]: It would be nice if it removed value alignment in let bindings

File: src/com/puppetlabs/http.clj [hunk 4]
```diff
@@ -231,7 +230,7 @@
    (streamed-response buffer
      ;; We pass the stream function back to the stream-query-result so that
      ;; it gets executed inside the cursor function.
```
Comment (ajroetker) [inline src/com/puppetlabs/http.clj]: It's weird these comments weren't aligned

**Pull request closed without merging**